### PR TITLE
Revert from hard coding back to a page query

### PR DESCRIPTION
**Pull request by dschapman**
@theianjones I know you've probably tried something similar. I reverted back to just using a page query and used the same syntax I use on my blog. It worked on develop and then I ran `gatsby build` and it worked then too. If this doesn't work then my best guess is that it's something on netlify's side that isn't building correctly.

### CODE REVIEW

**Comment by dschapman** [comment]
That's so weird it returns a completely empty array!

**Comment by dschapman** [comment]
Ian, this should fix it. **_Make sure to clear the cache on Netlify._** There's something with 1.0.13 of gatsby-theme-brain that seemed to have caused the issue. 

All I know is when I explicitly used 1.0.12 in develop mode it worked but when I used 1.0.13 it returned an empty array for the pagequery.

File: src/pages/articles.js [hunk 1]
```diff
@@ -4,308 +4,20 @@ import Layout from '../components/note-layout'
...
-          },
-        },
-      ],
-    },
-  },
-}
 
 function Articles(props) {
-  console.log({ props })
+  console.log(props.data )
```
Comment (theianjones) [inline src/pages/articles.js]:
```suggestion
```

**Review by theianjones — APPROVED** [review]
Thanks!

**Pull request merged**